godot-google-play-billing/src/main/java/org/godotengine/godot/plugin/googleplaybilling/GodotGooglePlayBilling.java

```diff
@@ -312,8 +326,8 @@ public Set<SignalInfo> getPluginSignals() {
 		signals.add(new SignalInfo("purchases_updated", Object[].class));
 		signals.add(new SignalInfo("query_purchases_response", Object.class));
 		signals.add(new SignalInfo("purchase_error", Integer.class, String.class));
-		signals.add(new SignalInfo("sku_details_query_completed", Object[].class));
-		signals.add(new SignalInfo("sku_details_query_error", Integer.class, String.class, String[].class));
+		signals.add(new SignalInfo("product_details_query_completed", Object[].class));
+		signals.add(new SignalInfo("product_details_query_error", Integer.class, String.class, String[].class));
```

Comment:
The last argument here is `String[]`, but below, it's called with `List<ProductDetails>`.